dspacAPI.py

```diff
@@ -0,0 +1,321 @@
+import asyncio
+import os
+import traceback
+from io import BytesIO
+
+from dotenv import load_dotenv
+
+from dspac_invest_api import DSPACAPI
+from helperAPI import (
+    Brokerage,
+    printAndDiscord,
+    printHoldings,
+    getOTPCodeDiscord,
+    maskString,
+    stockOrder,
+    send_captcha_to_discord,
+    getUserInputDiscord,
+)
+
+
+def dspac_init(DSPAC_EXTERNAL=None, botObj=None, loop=None):
+    load_dotenv()
+    dspac_obj = Brokerage("DSPAC")
+    if not os.getenv("DSPAC") and DSPAC_EXTERNAL is None:
+        print("DSPAC not found, skipping...")
+        return None
+
+    DSPAC = (
+        os.environ["DSPAC"].strip().split(",")
+        if DSPAC_EXTERNAL is None
+        else DSPAC_EXTERNAL.strip().split(",")
+    )
+    print("Logging in to DSPAC...")
+    for index, account in enumerate(DSPAC):
+        name = f"DSPAC {index + 1}"
+        try:
+            user, password, use_email = account.split(":")
+            use_email = use_email.upper()
+            ds = DSPACAPI(user, password, filename=f"DSPAC_{index + 1}.txt", creds_path="./creds/")
```

Comment:
Nelson Dane also mentioned this, but the creds file should be saved as a .pkl or json.